src/pages/Referrals/index.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <div class="referrals"></div>
+</template>
+
+<script setup lang="ts"></script>
+
+<style scoped lang="scss"></style>
```

Comment:
empty page?

Reply:
Yep, it's unfinished requst, it can be here.